frame/stake-tracker/migrations/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
```

Comment:
Future migrations of this pallet would probably live in itself, let's call this `stake-tracker-initializer` crate.

Reply:
done